dali/test/python/test_external_source_cupy.py

```diff
@@ -55,3 +55,22 @@ def generator(i):
     with check_output_pattern(pattern):
         for _ in range(iterations):
             pipe.run()
+
+
+def _test_cross_device(src, dst):
+    import nvidia.dali.fn as fn
```

Comment:
maybe move this import to the top?

Reply:
I don't know how it would impact the import order in the `impl` (if that's even important).